src/main/java/org/broadinstitute/hellbender/Main.java

```diff
@@ -117,24 +138,19 @@ protected final void mainEntry(final String[] args) {
             handleResult(result);
             System.exit(0);
         } catch (final CommandLineException e){
-            //the usage has already been printed so don't print it here.
-            if(printStackTraceOnUserExceptions()) {
-                e.printStackTrace();
-            }
+            handleUserException(e);
```

Comment:
Yes, that's currently the case, but it's going to be fixed soon I believe, with those exceptions being moved to a different exception type.